packages/clerk-js/src/ui/components/SessionTasks/tasks/TaskSelectOrganization/CreateOrganizationScreen.tsx

```diff
@@ -0,0 +1,170 @@
+              onAvatarChange={async file => setFile(file)}
+              onAvatarRemove={file ? onAvatarRemove : null}
+              actionTitle={localizationKeys('taskSelectOrganization.createOrganizationScreen.action__uploadAvatar')}
+              imageTitle={localizationKeys('taskSelectOrganization.createOrganizationScreen.avatarLabel')}
+              elementDescriptor={descriptors.organizationAvatarUploaderContainer}
+              avatarPreviewPlaceholder={
+                <IconButton
+                  variant='ghost'
+                  aria-label='Upload organization logo'
+                  icon={
+                    <Icon
+                      size='md'
+                      icon={Organization}
+                      sx={t => ({
+                        color: t.colors.$colorMutedForeground,
+                        transitionDuration: t.transitionDuration.$controls,
+                      })}
+                    />
+                  }
+                  sx={t => ({
+                    width: t.sizes.$16,
+                    height: t.sizes.$16,
+                    borderRadius: t.radii.$md,
+                    borderWidth: t.borderWidths.$normal,
+                    borderStyle: t.borderStyles.$dashed,
+                    borderColor: t.colors.$borderAlpha200,
+                    backgroundColor: t.colors.$neutralAlpha50,
+                    ':hover': {
+                      backgroundColor: t.colors.$neutralAlpha50,
+                      svg: {
+                        transform: 'scale(1.2)',
+                      },
+                    },
+                  })}
+                />
+              }
+            />
+          </Col>
+          <Form.ControlRow elementId={nameField.id}>
+            <Form.PlainInput
+              {...nameField.props}
+              onChange={onChangeName}
+              isRequired
+              autoFocus
+              ignorePasswordManager
+            />
+          </Form.ControlRow>
+          <Form.ControlRow elementId={slugField.id}>
+            <Form.PlainInput
+              {...slugField.props}
+              onChange={event => updateSlugField(event.target.value)}
+              isRequired
+              pattern='^(?=.*[a-z0-9])[a-z0-9\-]+$'
+              ignorePasswordManager
+            />
+          </Form.ControlRow>
+        </Flex>
+
+        <FormButtonContainer sx={t => ({ marginTop: t.space.$none, flexDirection: 'column' })}>
+          <Form.SubmitButton
+            block={false}
+            sx={() => ({ width: '100%' })}
+            isDisabled={isSubmitButtonDisabled}
+            localizationKey={localizationKeys('taskSelectOrganization.createOrganizationScreen.formButtonSubmit')}
+          />
+          {props.onCancel && (
+            <Form.ResetButton
+              localizationKey={localizationKeys('taskSelectOrganization.createOrganizationScreen.formButtonReset')}
+              block={false}
```

Comment:
Is the intention that this doesn't span full width with `block={false}`? This still spans full width due to being in a flex container.